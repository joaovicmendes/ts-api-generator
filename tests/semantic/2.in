// Deve reportar identificador n√£o declarado
model [
    Student {
        name: string
        email: string
        ra: number
        favorite_class: number
    }
    ExampleClass {
        student_ra: number
        grade: number
    }
]
endpoints [
    Student {
        GET '/students'
        GET '/students/:id'
        POST '/students'
        PUT '/students/:ra'
        DELETE '/students/:ra'
    }
    ExampleClass {
        GET 'class/:student_ra'
    }
    Class {
        GET 'class/'
    }
]
