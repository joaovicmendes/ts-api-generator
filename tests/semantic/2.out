Linha 17: identificador id não declarado anteriormente
Linha 25: identificador Class não declarado anteriormente
Compilação interrompida.